ably/realtime_conn.go

```diff
@@ -90,7 +90,7 @@ func newConn(opts *clientOptions, auth *Auth, callbacks connCallbacks) *Connecti
 		auth:      auth,
 		callbacks: callbacks,
 	}
-	auth.onExplicitAuthorize = c.onClientAuthorize
+	auth.onExplicitAuthorize = c.onExplicitAuthorize
```

Comment:
on the right side, `onExplicitAuthorize` makes sence, but on the left, it should be `authorizeRealtime`, the use-case here is very specific.